src/app/header/login-handler/user-card/user-card.component.ts

```diff
@@ -25,10 +25,25 @@ import { StartServiceService } from 'src/app/services/start/start-service.servic
 export class UserCardComponent implements OnInit {
   @Input('user') user: User
   showCard: boolean = false
+  photoUrl: string = ""
   constructor(private startService: StartServiceService, public authService: AuthService, public router: Router) { }
 
   ngOnInit(): void {
+    this.readProfileDetails();
```

Comment:
Why are we calling this method without any code inside it.

Reply:
removed it as it as unnecessary!